src/swbf_fnv_hashes.cpp

```diff
@@ -1,2834 +1,3935 @@
 
 #include "swbf_fnv_hashes.hpp"
 #include "synced_cout.hpp"
```

Comment:
[string_helpers.hpp](https://github.com/SleepKiller/swbf-unmunge/blob/master/src/string_helpers.hpp) has a fast function for converting an integer into a `0x0000` style string. It'll be better to use that rather than `sprintf` later.

```suggestion
#include "string_helpers.hpp"
#include "synced_cout.hpp"
```